upgrade/upgrade-scripts/upgrade-container

```diff
@@ -560,6 +580,7 @@ function migrate_configuration() {
 	while read -r file; do
 		migrate_file "$file"
 	done <<-EOF
+		/etc/default/nfs-kernel-server
```

Comment:
Why do we migration this file on upgrade?

Since we edit the file using delphix-platform, I think we can simply use the default contents for not-in-place upgrade, and then (again) rely on delphix-platform to properly modify it, right? This would help us get any changes made by the default version of the file, while also allowing us to make our "local" customizations on upgrade.

Reply:
We don't edit this file using delphix-platform. The virtualization code modifies the file as-needed at runtime (see http://reviews.delphix.com/r/50851/).

Reply:
> Upgrade code that copies the /etc/default/nfs-kernel-server file for not-in-place upgrades (because that file can be modified by the virtualization software as reviewed at http://reviews.delphix.com/r/50851).

Sorry, I see this was already explained. If we also modify this file via the virtualization software, then yes, we need to migrate this file like you're doing. I'll close this issue.